### PR TITLE
Fix UG, Add target user and value proposition

### LINKED ISSUE #103: [PE-D][Tester B] The target user for the product and/or the value proposition is not specified clearly.
The target user for the product and/or the value proposition is not specified clearly. 
I suspect it is for managing volunteers/elderly clients? but this is solely based off the feature commands.

Could be improved by adding some descriptions on who the product is for, and what the purpose is.

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: blaukc/ped#1

### LINKED ISSUE #105: [PE-D][Tester D] Removing Availability UG
Personally I found the wording of the caution a bit unclear. I recommend rephrasing it to "removing an availability will not delete the assignment tag linked to that availability"

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: myang2020/ped#1

### LINKED ISSUE #108: [PE-D][Tester B] ASSIGNMENTDETAILS is unclear & undefined
This can be due to he target user for the product and/or the value proposition is not specified clearly, as mentioned in a previous issue. This can also be solved by perhaps having adding it to an FAQ or having a glossary. 

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: blaukc/ped#5

### PR DESCRIPTION
**Pull request by tayziyi**
Fix #103 
Fix #105 
Fix #108

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -134,7 +138,7 @@ Format: `removeavail INDEX a/AVAILABILITY`
 
 <div markdown="block" class="alert alert-warning">
 :exclamation: **Caution:**
-In this version of Match, removing an availability will not delete his/her assignments on that day.
+In this version of Match, removing an availability will not delete assignments tagged to that deleted availability.
```
Comment (SimKianSeng) [inline docs/UserGuide.md:141]: perhaps can add a line saying `for record-keeping purposes`? Like to prevent being flagged as a feature flaw.

**Review by SimKianSeng — APPROVED** [review]
Minor notes, LGTM 👍🏻

File: docs/UserGuide.md (hunk 1 above)
Comment (Benson15912) [inline docs/UserGuide.md:141]: Maybe can move to another section for planned enhancements

**Review by Benson15912 — APPROVED** [review]
lgtm

**Pull request merged**